File: CCCanvas.Line.OC/CCCanvas.Line.OC/ViewController.m
//
//  ViewController.m
//  CCCanvas.Line.OC
//
//  Created by CC (deng you hua | cworld1000@gmail.com) on 2021/09/12.
//  https://github.com/ccworld1000/CCCanvas.Line

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (CCCanvasPoint *)getCanvasPoint {
	 return [[CCCanvasPoint alloc] initLinetWith2Point:CGPointMake(0.0, -1.0) pointB:CGPointMake(0.0, 1.0)];
}

- (void)viewDidLoad {
  [super viewDidLoad];
  // Do any additional setup after loading the view.
}


@end

File: CCCanvas.Line.Swift/CCCanvas.Line.Swift/ViewController.swift
//
//  ViewController.swift
//  CCCanvas.Line.Swift
//
//  Created by CC (deng you hua | cworld1000@gmail.com) on 2021/09/12.
//  https://github.com/ccworld1000/CCCanvas.Line

import UIKit
import CCCanvas

class ViewController: CCCanvasVC {

    override func getCanvasPoint() -> CCCanvasPoint {
        return CCCanvasPoint(linetWith2Point: CGPoint(x: 0, y: -1), pointB: CGPoint(x: 0, y: 1))
    }

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }
}


